src/app/pages/send/send-crypto-asset-form/stx/stx-crypto-currency-send-form.tsx

```diff
@@ -56,18 +68,27 @@ export function StxCryptoCurrencySendForm({}: StxCryptoCurrencySendFormProps) {
     nonce: nonceValidator,
   });
 
-  function onSubmit(values: any) {
-    logger.debug('stx submitted', values);
+  async function previewTransaction(values: any, formikHelpers: any) {
```

Comment:
👀

Reply:
Maybe I am missing what you mean here? You set up the `createDefaultInitialFormValues` which doesn't type the form values, right? I think I just didn't understand that each form needs it's own value type now, correct (like `SendFormValues`). I did setup the `ConfirmationRouteState` which is what I thought you were asking for. Sorry if I'm not understanding what you want here?

Reply:
I'm just pointing out the missing types. We know what the values types are when we type the form no? And doesn't formik export a helpers type for the last argument?

Reply:
`createDefaultInitialFormValues` creates the initial form values for each differently, so I can type each one individually. I did import the `FormikHelpers` type but needed the form values type as an arg so will fix both now that I understand what you mean. I just wasn't sure your intention with `createDefaultInitialFormValues`. I get it now.